src/dwarf.cpp

```diff
@@ -391,6 +294,124 @@ void line_header::read(freader& s, const file_details& details) {
 
 /**************************************************************************************************/
 
+attribute* alloc_attributes(std::size_t n) {
```

Comment:
This is the new routine that allocates 32MB blocks worth of `attribute` instances, and doles them out as necessary to `die`s that need them. Note the leaky memory switch, so they are *not* cleaned up at app shutdown.